plenum/server/catchup/catchup_rep_gatherer.py

```diff
@@ -17,6 +16,11 @@
 
 logger = getlogger()
 
+LedgerCatchupComplete = NamedTuple('LedgerCatchupComplete',
```

Comment:
I would prefer to have this message in `nodes_messages` because
- it's easier to find all the messages actors/services can exchange,
- if we are going to have more scalable and modular architecture, then it doesn't matter in principle if a message goes between nodes (in the network) or between local actors, since actors may be run in separate threads/processes/machines. So, every local message may become a network one.
- let's follow already defined approach that all messages are in one place until we re-factor/change it essentially for all messages if we want.
- additional static validation may help to follow the contract and catch exceptions and errors on CI phase